### PR TITLE
HW6_Vladyslav_Gurevych

### PR DESCRIPTION
**Pull request by VladyslavGurevych**

### CODE REVIEW

File: Vladyslav_Gurevych/HW6/Task1.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        operation.append(1)
+    elif i % 3 == 0:
+        operation.append(2)
+    else:
+        operation.append(3)
+
+for i in range(len(operation)):
+    line.append(f'{first_operand[i]} {second_operand[i]} {operation[i]}')
+
+result = tuple(line)
```
Comment (bohdandovbsyh) [inline Vladyslav_Gurevych/HW6/Task1.py]: it is tuple of strings, but I expect list of tuples

Comment (bohdandovbsyh) [reply Vladyslav_Gurevych/HW6/Task1.py]: image

File: Vladyslav_Gurevych/HW6/Task1_pickles.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        operation.append(1)
+    elif i % 3 == 0:
+        operation.append(2)
+    else:
+        operation.append(3)
+
+for i in range(len(operation)):
+    line.append(f'{first_operand[i]} {second_operand[i]} {operation[i]}')
+
+result = tuple(line)
```
Comment (bohdandovbsyh) [inline Vladyslav_Gurevych/HW6/Task1_pickles.py]: same here

File: Vladyslav_Gurevych/HW6/Task3.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import re
+
+with open("text.txt", 'r') as file:
+    text = file.read()
+
+text = re.findall('[A-z]', text)
+formatted_text = ''.join(text).lower()
+alphabet = "abcdefghijklmnopqrstuvwxyz"
```
Comment (bohdandovbsyh) [inline Vladyslav_Gurevych/HW6/Task3.py:8]: alternative way:

import string
alphabet = string.ascii_lowercase

File: Vladyslav_Gurevych/HW6/Task3.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    result[char] = 0
+
+for char in formatted_text:
+    for key, value in result.items():
+        if char == key:
+            value = value + 1
+            result[char] = value
+
+for key, value in result.items():
+    print(f'Letter "{key}" appears {value} times in this text.')
```
Comment (bohdandovbsyh) [inline Vladyslav_Gurevych/HW6/Task3.py:20]: Good. Alternative solution:

import re
with open("text.txt", 'r') as file:
    text = file.read()
regex = re.compile('[^A-z]')
result = list(regex.sub('', text).lower())
resulted_dict = {}.fromkeys(result)
for key in resulted_dict.keys():
    resulted_dict[key] = result.count(key)
print(resulted_dict)

**Review by bohdandovbsyh — APPROVED** [review]

File: Vladyslav_Gurevych/HW6/Task1.py (hunk 1 above)
Comment (VladyslavGurevych) [reply Vladyslav_Gurevych/HW6/Task1.py]: fixed

File: Vladyslav_Gurevych/HW6/Task1_pickles.py (hunk 2 above)
Comment (VladyslavGurevych) [reply Vladyslav_Gurevych/HW6/Task1_pickles.py]: fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**